src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/ProjectSystemCache.cs

```diff
@@ -268,7 +270,7 @@ public bool AddProject(ProjectNames projectNames, IVsProjectAdapter vsProjectAda
             return true;
         }
 
-        public bool AddProjectRestoreInfo(ProjectNames projectNames, DependencyGraphSpec projectRestoreInfo)
+        public bool AddProjectRestoreInfo(ProjectNames projectNames, DependencyGraphSpec projectRestoreInfo, IReadOnlyList<IAssetsLogMessage> additionalMessages)
```

Comment:
Public method - consider an overload?

Reply:
As the other comment, this is not an assembly that we provide in a package.